Common subdirectories: pkgdiff-1.7.2/doc and pkgdiff-1.7.2-zst/doc
Common subdirectories: pkgdiff-1.7.2/modules and pkgdiff-1.7.2-zst/modules
diff -u pkgdiff-1.7.2/pkgdiff.pl pkgdiff-1.7.2-zst/pkgdiff.pl
--- pkgdiff-1.7.2/pkgdiff.pl	2016-03-18 12:29:58.000000000 +0000
+++ pkgdiff-1.7.2-zst/pkgdiff.pl	2020-01-09 23:03:16.270580393 +0000
@@ -461,11 +461,14 @@
 
     "TAR.LZ"   => ["tar.lz", "tlz"],
 
+    "TAR.ZST"  => ["tar.zst", "tzst"],
+
     "ZIP"      => ["zip", "zae"],
     "TAR"      => ["tar"],
     "LZMA"     => ["lzma"],
     "GZ"       => ["gz"],
     "XZ"       => ["xz"],
+    "ZST"      => ["zst"],
 
     "JAR"      => ["jar", "war",
                    "ear"]
@@ -2903,6 +2906,9 @@
     elsif($Format eq "XZ") {
         $Cmd = "cp -f \"$Pkg\" \"$OutDir\" && cd \"$OutDir\" && unxz \"".get_filename($Pkg)."\"";
     }
+    elsif($Format eq "ZST") {
+        $Cmd = "cp -f \"$Pkg\" \"$OutDir\" && cd \"$OutDir\" && unzstd --rm \"".get_filename($Pkg)."\"";
+    }
     elsif($Format eq "ZIP") {
         $Cmd = "unzip -o \"$Pkg\" -d \"$OutDir\"";
     }
